src/backend/services/docker_service.py

```
@@ -1,20 +1,82 @@
+    for container_ref in candidate_container_refs:
+        if not container_ref:
+            continue
+        try:
+            current_container = client.containers.get(container_ref)
+            for mount in current_container.attrs.get('Mounts', []):
+                if mount.get('Destination') == '/app/robot_tests' and mount.get('Source'):
+                    source = mount['Source']
+                    logging.info(
+                        f"🐳 DOCKER SERVICE: Resolved host robot_tests mount from container inspect ({container_ref}): {source}")
+                    return source
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)